test/app/features/feed/domain/usecases/tweet_toggle_feature_test.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:penhas/app/core/managers/modules_sevices.dart';
+import 'package:penhas/app/features/appstate/domain/entities/app_state_entity.dart';
+import 'package:penhas/app/features/feed/domain/usecases/tweet_toggle_feature.dart';
+
+void main() {
+  late MockAppModulesServices modulesServices;
+  late TweetToggleFeature feature;
+
+  setUp(() {
+    modulesServices = MockAppModulesServices();
+    feature = TweetToggleFeature(modulesServices: modulesServices);
+  });
+
+  group(TweetToggleFeature, () {
+    group('isEnabled', () {
+      test('returns true when feature with featureCode is present', () async {
+        when(() => modulesServices.feature(name: any(named: 'name')))
```

Comment:
Talvez seria uma boa garantir que o nome da feature está sendo passado corretamente, ou talvez criar um caso de teste com um verify

```suggestion
        when(() => modulesServices.feature(name: 'tweets'))
```

Reply:
> Talvez seria uma boa garantir que o nome da feature está sendo passado corretamente, ou talvez criar um caso de teste com um verify

Melhor validar isto no `verify`, assim desta maneira é possível identificar o motivo da falha. Se colocar isto no when vamos receber uma mensagem de `null` que vai dificultar o `debug` da falha no teste.